resources/views/layouts/app.blade.php

```diff
@@ -67,16 +64,6 @@
                                 </li>
                             @endif
                         @else
```

Comment:
this was remove but this part https://github.com/kredo-school/14th_explore_ease/blob/9564049fdadf44cf3b7864dd6cf07f8173004e15/routes/web.php#L33 is still present

Reply:
why are we removing this again?